language/diem-framework/modules/0L/AutoPay.move

```diff
@@ -75,10 +75,9 @@ address 0x1 {
     // It keeps track of all accounts that have autopay enabled and updates the 
     // list as accounts change their Status structs
 
-    // current_epoch value is unused and should be ignored
     struct AccountList has key {
       accounts: vector<address>,
-      current_epoch: u64, //unused
+      // current_epoch: u64, // todo: unused, delete?
```

Comment:
Ideally yes, delete, but I'm concerned this will cause issues with state migration. I don't know what the migration strategy is though, so maybe its fine.